src/scripts/main.js

```diff
@@ -21,7 +21,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  element.appendChild(ul);
+
+  for (const [key, value] of Object.entries(data)) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(value).length !== 0) {
```

Comment:
it works, but note that if the value was null or undefined it would cause error
So it's always better if it's an object before doing `Object.keys`

FYI